### PR TITLE
ppc: Add libglm

### PR DESCRIPTION
**Pull request by QuarkTheAwesome**
Hi again!
Another powerpc library; this one is [glm](https://github.com/g-truc/glm). This is adapted from the Switch version, everything seems to work fine! Hopefully I didn't stuff up cmake again - I needed to add `-DCMAKE_SYSTEM_NAME` since devkitppc.cmake doesn't set that and I was reluctant to use wii.cmake or whatever. Leave it alone and it'll force `-rdynamic` into the link flags...
In any case, this works well.
Thanks!

### CODE REVIEW

File: ppc/glm/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  cd build
+
+  source $DEVKITPRO/ppcvars.sh
+
+  cmake -G"Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=/opt/devkitpro/devkitppc.cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_SYSTEM_NAME=Generic \
+    -DCMAKE_INSTALL_PREFIX=$PORTLIBS_PREFIX \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_CXX_FLAGS="$CFLAGS $CPPFLAGS -D_GLIBCXX_USE_C99_MATH_TR1 -D_LDBL_EQ_DBL" \
```
[the remark this answers is not in the document]
Comment (QuarkTheAwesome) [reply ppc/glm/PKGBUILD]: Thanks for that, you're right. Fixing in c6cb7fb.

**Comment by QuarkTheAwesome** [comment]
Bumping the version down to 0.9.9.2 because of issues like [this one](https://github.com/g-truc/glm/issues/832).

File: ppc/glm/PKGBUILD (hunk 1 above)
[the remark this answers is not in the document]
Comment (QuarkTheAwesome) [reply ppc/glm/PKGBUILD]: Oops! Is 63066ca okay?

**Comment by WinterMute** [comment]
> Bumping the version down to 0.9.9.2 because of issues like [this one](https://github.com/g-truc/glm/issues/832).

Are there more issues? We're using gcc 8.2.0 now which defaults to gnu++14

**Comment by QuarkTheAwesome** [comment]
My test app specifies gnu++11 in the makefile, so even on the latest toolchain it doesn't compile with 0.9.9.3. Using 0.9.9.2, however, it works great! Not aware of any other issues with it.

**Comment by fincs** [comment]
Why do you have gnu++11 in your makefile? Any reason why not just remove it?

**Comment by WinterMute** [comment]
Depending on what the app is, you could submit a PR to it.

I think it's probably reasonable to set CMAKE_SYSTEM_NAME in devkitppc.cmake tbh. These cmake toolchain files are still very much a work in progress

**Comment by WinterMute** [comment]
I made the CMAKE_SYSTEM_NAME change in the pkgbuild scripts package. https://github.com/devkitPro/pacman-packages/commit/cf631cebc1d1faa684f1ca9eea04765498c0632e Can you try with out that line in your cmake configure now?

**Comment by QuarkTheAwesome** [comment]
Working great with `dkp-libs/devkitpro-pkgbuild-helpers 1.2.2-1`! bfe6caf removes CMAKE_SYSTEM_NAME.

**Pull request merged**